### ISSUE TITLE
Timetable is crashing the app and showing nothing

### ISSUE THREAD

**Issue by Baumlaeufer0815** [koenidv/sph-planner#37]
**Describe the bug**
If I press the "🏠 " Button or the "<" and ">" Buttons, the app will crash and the timetable is showing nothing, no courses or whatever. 

**I get this Error Message, if I press the house button**, the other error messages of the other buttons are almost identical
```
E/AndroidRuntime: FATAL EXCEPTION: main
    Process: de.koenidv.sph, PID: 22797
    java.lang.NullPointerException: null cannot be cast to non-null type de.koenidv.sph.adapters.TimebarAdapter
        at de.koenidv.sph.ui.TimetableViewFragment$onCreateView$3.onClick(TimetableViewFragment.kt:275)
        at android.view.View.performClick(View.java:7448)
        at com.google.android.material.button.MaterialButton.performClick(MaterialButton.java:1119)
        at android.view.View.performClickInternal(View.java:7425)
        at android.view.View.access$3600(View.java:810)
        at android.view.View$PerformClick.run(View.java:28305)
        at android.os.Handler.handleCallback(Handler.java:938)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:223)
        at android.app.ActivityThread.main(ActivityThread.java:7656)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
```
I am using a simulated Pixel 2 with API 30 and my school is "Otto-Hahn-Schule, Frankfurt a. M. (6091)"

![screenshot](https://user-images.githubusercontent.com/73837560/150689414-a0db7d76-8dc1-47b8-ab45-695a9d01e7be.png)

**Comment by Tiefkhlpizza** [comment]
Hi Baumlaeufer0815,

I forked the project to develop some stuff in a branch. I was aware of this crash but did not expect that someone else is already using my branch. Nevertheless, tdy I updated several things in my branch. 

Now this crash should be solved because switching the weeks is only allowed in home fragment anymore. If you go in the timetable view, these buttons are not visible anymore.

Best regards
Stefan
